2.6/content_sets.yml

```diff
@@ -1,10 +1,21 @@
-# This is a file defining which content sets are needed to update content in 
+# This is a file defining which content sets are needed to update content in
 # this image. Data provided here helps determine which images are vulnerable to
 # specific CVEs. Generally you should only need to update this file when:
 #    1. You start depending on new product
 #    2. You are preparing new product release and your content sets will change
 ---
 x86_64:
-- rhel-7-server-rpms
-- rhel-7-server-optional-rpms
-- rhel-server-rhscl-7-rpms
+- rhel-8-for-x86_64-baseos-rpms
+- rhel-8-for-x86_64-appstream-rpms
+
+ppc64le:
+- rhel-8-for-ppc64le-baseos-rpms
+- rhel-8-for-ppc64le-appstream-rpms
+
+s390x:
+- rhel-8-for-s390x-baseos-rpms
+- rhel-8-for-s390x-appstream-rpms
+
+aarch64:
+- rhel-8-for-aarch64-baseos-rpms
+- rhel-8-for-aarch64-appstream-rpms
```

Comment:
I dont think we need to change this file. Tbh I do not see any reason to keep it in upstream at all